src/libais/ais.h

```diff
@@ -373,6 +373,16 @@ enum Dac {
 
 class AisBitset;
 
+class AisMdate
```

Comment:
I'm not sure this class adds alot.  For starters, the class name doesn't mean anything to me.  Perhaps a better class name and definitely needs a comment.